README.md

```diff
@@ -237,51 +195,48 @@ Example:
 
 Contributions, issues, and feature requests are welcome!
 
-Feel free to check the [issues page](../../issues/).
-
+Feel free to check the [issues page](https://github.com/belaymit/math-magicians/issues).
+..
```

Comment:
- Please, let's update the link to the issues page to be the current project as this points to another project.